tools/device_configuration_checker.py

```diff
@@ -299,35 +300,38 @@ def _run_tests_on_know_good_image(args, device):
 
     image = None
     if device.model.lower() == "beagleboneblack":
-        image = image_directory_path
+        image = "good-image.tar.bz2"
     elif device.model.lower() == "edison":
-        image = os.path.join(
-            image_directory_path,
-            "ostro-image-edison.ext4")
+       image = "good-image.ext4"
```

Comment:
I would prefer a "fail if not explicitly matching" approach - adding a bit more code for new cases shouldn't be too bad. Or is it?